build-support/editor_setup.py

```diff
@@ -42,6 +42,8 @@ class PyrightConfig(TypedDict):
     "exclude": [
         "src/js/**",
         "src/rust/**",
+        "bin/**",
+        "dist/**",
```

Comment:
pyright was very slow, was trying to figure out why. I thought it might be scanning the zipfiles in these directories so I added them to the ignore list. Will remove.